Ugh... getting this one again. Something's flaky in CI; looks like some potential issue with some action;

```
Run .\hack\make.ps1 -TestIntegration
INFO: make.ps1 starting at 01/31/2024 13:32:31
template: main:1: unterminated quoted string
INFO: make.ps1 ended at 01/31/2024 13:32:32
Error: Process completed with exit code 1.
```

--- a/Dockerfile.windows
+++ b/Dockerfile.windows
@@ -168,7 +168,7 @@ SHELL ["powershell", "-Command", "$ErrorActionPreference = 'Stop'; $ProgressPref
 ARG GO_VERSION=1.20.13
 ARG GOTESTSUM_VERSION=v1.8.2
 ARG GOWINRES_VERSION=v0.3.0
-ARG CONTAINERD_VERSION=v1.6.22
+ARG CONTAINERD_VERSION=v1.6.27
 
 # Environment variable notes:
 #  - GO_VERSION must be consistent with 'Dockerfile' used by Linux.

